xds/src/main/java/io/grpc/xds/XdsLoadBalancer.java

```diff
@@ -89,18 +89,23 @@ public void onError() {
   @Override
   public void handleResolvedAddressGroups(
       List<EquivalentAddressGroup> servers, Attributes attributes) {
-    Map<String, Object> newLbConfig = checkNotNull(
+    Map<String, Object> newRawLbConfig = checkNotNull(
         attributes.get(ATTR_LOAD_BALANCING_CONFIG), "ATTR_LOAD_BALANCING_CONFIG not available");
-    fallbackPolicy = selectFallbackPolicy(newLbConfig, lbRegistry);
-    fallbackManager.updateFallbackServers(servers, attributes, fallbackPolicy);
-    fallbackManager.maybeStartFallbackTimer();
-    handleNewConfig(newLbConfig);
-    xdsLbState.handleResolvedAddressGroups(servers, attributes);
+    try {
+      LbConfig newLbConfig = ServiceConfigUtil.unwrapLoadBalancingConfig(newRawLbConfig);
+      fallbackPolicy = selectFallbackPolicy(newLbConfig, lbRegistry);
+      fallbackManager.updateFallbackServers(servers, attributes, fallbackPolicy);
+      fallbackManager.maybeStartFallbackTimer();
+      handleNewConfig(newLbConfig);
+      xdsLbState.handleResolvedAddressGroups(servers, attributes);
+    } catch (Exception e) {
+      // TODO: handle it
```

Comment:
Sorry, this is not acceptable.

Reply:
I have changed it to rethrowing so it's keeping the current behavior.